frontend/mono-ui/web/rainmaker/packages/employee/src/modules/employee/Erp/EGF/index.js

```diff
@@ -28,8 +28,13 @@ class EGFFinance extends Component {
     // Reading environment name (ex: dev, qa, uat, fin-uat etc) from the globalconfigs if exists else reading from the .env file
     finEnv = this.globalConfigExists() ? window.globalConfigs.getConfig("FIN_ENV") : process.env.REACT_APP_FIN_ENV;
     // Preparing finance subdomain url using the above environment name and the domain url
-    subdomainurl = !!(finEnv) ? "-" + finEnv + "." + domainurl : "." + domainurl;
-    erp_url = loc.protocol + "//" + getTenantId().split(".")[1] + subdomainurl + menuUrl;
+    // For deployed project:
```

Comment:
Why you are committing this change, this change is meant only for local development.